### PR TITLE
missing scholarrank profs (250+)

### PR DESCRIPTION
**Pull request by leonidk**
In the same style as #1268, added faculty at existing CSRankings universities that were in scholarrank's results. Did some manual filtering to remove non-CS profs, etc. but I likely missed many of them still. There's likely some errors of the nickname variety leading to an error in DBLP affiliation but these websites all seemed accurate at the time. 

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -14625,6 +14625,282 @@ Yvonne Jansen,University of Copenhagen,http://www.diku.dk/Ansatte/?pure=da%2Fper
...
+Michal Cutler,Binghamton University,http://www.cs.binghamton.edu/~cutler/,NOSCHOLARPAGE
+Ping Yang,Binghamton University,http://www.cs.binghamton.edu/~pyang/,NOSCHOLARPAGE
+Yan Wang,Binghamton University,http://www.cs.binghamton.edu/~yanwang/,NOSCHOLARPAGE
+Yi-Fan Zhang,Binghamton University,http://www.cs.binghamton.edu/~zhangy/,Mi5cKMkAAAAJ
+Gary Benson,Boston University,http://tandem.bu.edu/benson.html,E0GO0JkAAAAJ
+Christophe Giraud,Brigham Young University,http://dml.cs.byu.edu/,jHWj2yYAAAAJ
+Casey T. Deccio,Brigham Young University,https://casey.byu.edu/,NOSCHOLARPAGE
+Frank Jones,Brigham Young University,https://marriottschool.byu.edu/directory/details,NOSCHOLARPAGE
+Sherief Reda,Brown University,https://vivo.brown.edu/display/sreda,p69MQp4AAAAJ
+James Tompkin,Brown University,https://cs.brown.edu/courses/csci1430/,Ex3pgLAAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:14661]: Correct home page should be: http://jamestompkin.com/

File: csrankings.csv [hunk 2]
```diff
@@ -14625,6 +14625,282 @@ Yvonne Jansen,University of Copenhagen,http://www.diku.dk/Ansatte/?pure=da%2Fper
...
+Ming-Chun Huang,Case Western Reserve University,https://engineering.case.edu/about/school-directory/ming-chun-huang,NOSCHOLARPAGE
+Gregory S. Lee,Case Western Reserve University,http://engineering.case.edu/profiles/gsl7,NOSCHOLARPAGE
+Jim F. Martin,Clemson University,https://www.clemson.edu/madren/inn/,rUUtArEAAAAJ
+Xu Liu,College of William and Mary,http://www.cs.wm.edu/~xl10/,qhLT67EAAAAJ
+Bo Wu,Colorado School of Mines,http://inside.mines.edu/~bwu/,g6Wdt1YAAAAJ
+Adele E. Howe,Colorado State University,http://www.cs.colostate.edu/~howe/,FjV70BwAAAAJ
+Laura Moreno,Colorado State University,https://www.cs.colostate.edu/~lmorenoc/,UJM5NfoAAAAJ
+Louis-Noël Pouchet,Colorado State University,http://www.cs.colostate.edu/~pouchet/,TCppIZYAAAAJ
+Xi Chen,Columbia University,http://www.cs.columbia.edu/~xichen/Homepage/Welcome.html,y2pH4jcAAAAJ
+Angelos D. Keromytis,Columbia University,http://www.cs.columbia.edu/~angelos/,mncyWbcAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:14674]: In fact, he's at GAtech now (left DARPA). https://www.ece.gatech.edu/faculty-staff-directory/angelos-d-keromytis

File: csrankings.csv [hunk 3]
```diff
@@ -14625,6 +14625,282 @@ Yvonne Jansen,University of Copenhagen,http://www.diku.dk/Ansatte/?pure=da%2Fper
...
+Alla Rozovskaya,CUNY,https://www.gc.cuny.edu/Page-Elements/Academics-Research-Centers-Initiatives/Doctoral-Programs/Computer-Science/Faculty-Bios/Alla-Rozovskaya,SL4bZHEAAAAJ
+Huy T. Vo,CUNY,https://serv.cusp.nyu.edu/~hvo/,HNwwWQgAAAAJ
+Qiang Liu,Dartmouth College,https://web.cs.dartmouth.edu/people/qiang-liu,XEx1fZkAAAAJ
+Bruce Donald,Duke University,http://www.cs.duke.edu/brd/,TkayRqQAAAAJ
+Kishor S. Trivedi,Duke University,https://trivedi.pratt.duke.edu/,fEVXhhAAAAAJ
+Jun Yang,Duke University,https://users.cs.duke.edu/~junyang/,NOSCHOLARPAGE
+Landon P. Cox,Duke University,https://users.cs.duke.edu/~lpcox/,zoBGvW4AAAAJ
+Benjamin Lee,Duke University,http://people.duke.edu/~bcl15/,4Tnj6PcAAAAJ
+Sina Farsiu,Duke University,http://people.duke.edu/~sf59/,mzcr92sAAAAJ
+Rong Ge,Duke University,https://users.cs.duke.edu/~rongge/,MVxcjEoAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:14689]: Already in the database as:
* Rong Ge 0001,Duke University,https://users.cs.duke.edu/~rongge,MVxcjEoAAAAJ

File: csrankings.csv [hunk 4]
```diff
@@ -14625,6 +14625,282 @@ Yvonne Jansen,University of Copenhagen,http://www.diku.dk/Ansatte/?pure=da%2Fper
...
+Amy Williams,Cornell University,http://williamslab.bscb.cornell.edu/,_-PAsvMAAAAJ
+Haiyuan Yu,Cornell University,http://yulab.icmb.cornell.edu/,NOSCHOLARPAGE
+Lei Xie,CUNY,http://compsci.hunter.cuny.edu/~leixie/,CujCxdYAAAAJ
+Alla Rozovskaya,CUNY,https://www.gc.cuny.edu/Page-Elements/Academics-Research-Centers-Initiatives/Doctoral-Programs/Computer-Science/Faculty-Bios/Alla-Rozovskaya,SL4bZHEAAAAJ
+Huy T. Vo,CUNY,https://serv.cusp.nyu.edu/~hvo/,HNwwWQgAAAAJ
+Qiang Liu,Dartmouth College,https://web.cs.dartmouth.edu/people/qiang-liu,XEx1fZkAAAAJ
+Bruce Donald,Duke University,http://www.cs.duke.edu/brd/,TkayRqQAAAAJ
+Kishor S. Trivedi,Duke University,https://trivedi.pratt.duke.edu/,fEVXhhAAAAAJ
+Jun Yang,Duke University,https://users.cs.duke.edu/~junyang/,NOSCHOLARPAGE
+Landon P. Cox,Duke University,https://users.cs.duke.edu/~lpcox/,zoBGvW4AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:14686]: Landon left Duke (permanently) for MSR.

File: csrankings.csv [hunk 5]
```diff
@@ -14625,6 +14625,282 @@ Yvonne Jansen,University of Copenhagen,http://www.diku.dk/Ansatte/?pure=da%2Fper
...
+Ruogu Fang,Florida International University,https://www.bme.ufl.edu/labs/fang/ruogu/index.html,LVb46zEAAAAJ
+Sam Ganzfried,Florida International University,https://www.cis.fiu.edu/professor-sam-ganzfried-featured-miami-herald/,gnw1BGkAAAAJ
+Radu Jianu,Florida International University,http://digitalcommons.fiu.edu/etd/2566/,z1YqlisAAAAJ
+Ning Xie,Florida International University,http://users.cis.fiu.edu/~nxie/,H9x5YXkAAAAJ
+Xin Yuan,Florida State University,http://www.cs.fsu.edu/~xyuan/,3pkXlIEAAAAJ
+Viet Tung Hoang,Florida State University,http://www.cs.fsu.edu/~tvhoang/,ggcCMuIAAAAJ
+Zhi Wang,Florida State University,http://www.cs.fsu.edu/~zwang/,Y9SwH_0AAAAJ
+Jie Yang,Florida State University,http://www.cs.fsu.edu/~jieyang/,8e6Jnq4AAAAJ
+Mark Borodovsky,Georgia Institute of Technology,https://www.cc.gatech.edu/people/mark-borodovsky,ciQ3dn0AAAAJ
+Frank Dellaert,Georgia Institute of Technology,https://www.cc.gatech.edu/people/frank-dellaert,ZxXBaswAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:14702]: Correct home page:
* https://www.cc.gatech.edu/~dellaert/FrankDellaert/Frank_Dellaert/Frank_Dellaert.html

**Review by emeryberger — CHANGES_REQUESTED** [review]
This review is the result of a spot check. I found lots of issues at a glance. I think it would make a lot of sense and be a lot easier and more auditable with separate PRs for each institution.

**Comment by leonidk** [comment]
Yeah, I'd like to do a manual audit of this list if I get some extra time (the names from the dataset were often bad/misspelled/missing middle names, and certainly had no DBLP disambiguation). I used some fuzzy string matching to get DBLP names and a scraper to get websites. But I figured I'd open a pull request so (at least) this initial set is available to people. The names were ordered by university to make auditing easier but ultimately it'll just require a chunk of time from someone, and I'm not sure when & if that someone could be me. 

I think reaching out to [these authors](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0202223) for the guids <-> names in their [dataset](https://github.com/allisonmorgan/academic_census) might be useful.

**Comment by emeryberger** [comment]
I don't have the time to personally audit this. Can you resolve the changes I mentioned and then break this up into commits per school so I (and others) can review these? Otherwise, I will have to close this. Thanks.

**Comment by emeryberger** [comment]
bump @leonidk . I hate to discard all these but I cannot manually audit such a large commit.

**Comment by emeryberger** [comment]
Closing for lack of response.

**Pull request closed without merging**